### ISSUE TITLE
Support for nodejs v16 without deprecation notice

### ISSUE THREAD

**Issue by doberkofler** [stylelint/stylelint#5260]
It would be nice to get rid of the following deprecation notice when running on nodejs v 16

```
(node:61490) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at /Users/doberkofler/MyDev/ljs_app/trunk/periscope/node_modules/postcss/package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
```

Most likely related to https://github.com/postcss/postcss/issues/1561

**Comment by jeddy3** [comment]
I'm unable to replicate this locally using nodejs v16.

We using PostCSS@7 in stylelint at the moment, and [that doesn't use the `exports` field](https://github.com/postcss/postcss/blob/v7/package.json).

Can you provide steps to reproduce, please?

**Comment by doberkofler** [comment]
The problem has been resolved by using postcss 8.2.12
